openmetadata-docs/content/v1.3.x/developers/contribute/build-code-and-run-tests/openmetadata-server.md

```diff
@@ -103,6 +103,16 @@ In that list look for "jersey-client:2.25.1"
 
 Select it and click "OK". 
 
+Click on Module Dependency
+
+{% image src="/images/v1.3/developers/contribute/build-code-and-run-tests/intellij-library.png" alt="Click on Library" caption=" " /%}
+
+In that list look for "openmetadata-ui"
+
+{% image src="/images/v1.3/developers/contribute/build-code-and-run-tests/intellij-Openmetadata-UI.png" alt="Add openmetadata-ui dependency" caption=" " /%}
+
+Select it and click "OK". 
+
```

Comment:
<img width="1302" alt="Screenshot 2024-05-09 at 8 48 08 AM" src="https://github.com/open-metadata/OpenMetadata/assets/40225091/67c5ea82-1d21-4985-a9cf-337f037e0063">
Let's modify the image instead of adding separate note for openmetadata-ui